docs/docs/user-guide/permissions.md

```diff
@@ -0,0 +1,123 @@
+
+- **Create Roles**: You can create new server-level roles from this section. You can configure these roles in two ways:
+    1. With login capability and a password, which you can assign to collaborators.
+    2. Without login capability, to be used exclusively as a parent role to group permissions that can be inherited by other roles. You cannot assign these non-login roles to collaborators directly.
+- **Define Child Roles**: PostgreSQL has a mechanism for [Role Membership](https://www.postgresql.org/docs/current/role-membership.html) wherein any role can be "granted" to any other role to form simple hierarchies or complex graph-based inheritance structures. For any role you've configured within Mathesar, you can use Mathesar to grant the role to other "child roles".
+- **Drop Roles**: You can drop server-level roles that are no longer needed. This action removes the role from the server, however if the role is configured in Mathesar, it will still be displayed. Exercise caution when dropping roles, as it may affect collaborators using the dropped role in Mathesar.
+
+!!! note
+    Server roles, once added, must be configured in Mathesar under the **Role Configuration** section before they can be assigned to collaborators.
+
+---
+
+## PostgreSQL objects {:#objects}
+
+In PostgreSQL, an "object" is a thing like: a database, a schema, a table, _(and some other things too, which we won't cover here)_.
+
+### Privileges and ownership
+
+- **Privileges:** Specific privileges on an object can be granted to specific roles.
+
+    !!! example
+        A role can be granted the `CREATE` privilege on a schema. This allows the role to create new tables within the schema.
+
+- **Ownership**: Every PostgreSQL object has one and only one role said to be its "owner". The owner generally can do anything directly to the object, but not necessarily other objects contained within it. By default the owner is set to the role which created the object.
+
+- **Shared ownership:** While PostgreSQL has a variety of granular privileges for different actions, there are still certain actions which remain restricted to object owners. For example only the owner of a table can add new columns to it.
+
+    While this behavior may seem limiting, it's still possible configure multiple roles to effectively "own" a single object by leveraging PostgreSQL's powerful role inheritance functionality:
+    
+    1. We can create a third role to directly own the object and act as a sort of proxy "group". (The group role doesn't need to be a `LOGIN` role and thus doesn't require a password to be configured.)
+    1. Then we can grant _that group role_ to any other roles we'd like.
+    1. Those granted roles will then have permission do things _as if they were the owner themselves_.
```

Comment:
There is no instruction on how to do this via our UI. Do we add them?

We call them 'child roles' on the UI.

Reply:
Step-by-step instructions for configuring shared ownership would probably be useful. But I don't think it's high enough priority to add right now.

I pushed e7fc8e142fc10b6995e169b977b524c5243748e8 to change "granted roles" to "child roles". Thanks for that suggestion.